evergreen.yml

```
@@ -301,6 +303,28 @@ functions:
         permissions: public-read
         content_type: application/octet-stream
 
+  "update download center feed":
+    - command: shell.exec
+      params:
+        shell: bash
+        working_dir: mongo-powerbi-connector
+        script: |
+          ${prepare_shell}
+          sed -i 's@{RELEASE_VERSION}@'${RELEASE_VERSION}'@' mongo-powerbi-connector-downloads_template.json
+          sed -i 's@{RELEASE_PQX_PATH}@'${RELEASE_PQX_PATH}'@' mongo-powerbi-connector-downloads_template.json
+          echo "-------------------------"
+          cat mongo-powerbi-connector-downloads_template.json
+          echo "-------------------------"
+    - command: s3.put
+      params:
+        aws_key: ${release_aws_key}
+        aws_secret: ${release_aws_secret}
+        local_file: mongo-tableau-connector/mongo-powerbi-connector-downloads_template.json
+        remote_file: mongo-tableau-connector/mongo-powerbi-connector-downloads.json
```

I'm not sure if all the changes propagated, but I'm still seeing two instances of `tableau`.

```suggestion
        local_file: mongo-powerbi-connector/mongo-powerbi-connector-downloads_template.json
        remote_file: mongo-powerbi-connector/mongo-powerbi-connector-downloads.json
```